src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagEntry.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag
+{
+    /// <summary>
+    /// External representation of a extended query tag entry for add.
+    /// </summary>
+    public class AddExtendedQueryTagEntry : ExtendedQueryTagEntry, IValidatableObject
+    {
+        /// <summary>
+        /// Level of this tag. Could be Study, Series or Instance.
+        /// </summary>
+        public string Level { get; set; }
```

Comment:
At this moment, I'm thinking if _QueryTagLevel?_  could be a better choice, seems it could simplify things a lot